seed_data/management/commands/create_tiers.py

```diff
@@ -20,6 +20,20 @@ def create_tiers(programs, num_tiers):
         programs_with_tiers_created += 1
         created_tier_programs = TierProgramFactory.create_batch(tiers_to_create_count, program=program)
         tiers_created += len(created_tier_programs)
+
+        # there must be at least one TierProgram with discount_amount=0 and one with income_threshold=0
+        least_income_threshold = sorted(
+            created_tier_programs, key=lambda tier_program: tier_program.income_threshold
```

Comment:
why not inplace sorting. 2 blocks ``least_income_threshold`` and ``least_discount`` seems independent.?